Andrii_Voinenko/Lesson5/task3.py

```diff
@@ -0,0 +1,27 @@
+import re
+info = "name=Amanda=sssss&age=32&&salary=1500&currency=euro"
+
+update_info = re.split("\W", info)
+values_list = []
+
+for element in update_info:
+    if element == 'sssss':
+        update_info.remove(element)
+    elif element == '':
+        update_info.remove(element)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/188333512-768b8399-e3b3-4b4e-9998-e2cd9a9199d6.png)
This variant will be better